python/setup.py

```diff
@@ -140,7 +153,7 @@ def get_instantiation_cu() -> list[str]:
 def get_version():
     version = os.getenv("FLASHINFER_BUILD_VERSION")
     if version is None:
-        with open(root / "version.txt") as f:
+        with open(root / "../version.txt") as f:
```

Comment:
Can you try resolve the soft link?
```suggestion
        with open((root / "version.txt").resolve()) as f:
```

Reply:
I know why this isn't work for me. Because i downloaded this repo using zip file and soft links are broken.😰 I will revert this change later. Thanks!